src/arduino_iot_cloud/__init__.py

```diff
@@ -29,6 +29,7 @@
     b"f612f28123ee92d55b5bfb11d7ecec3f86c5a767"
 )
 
+
```

Comment:
Please squash this commit.

Also note the CI is failing because commit messages end with ".". They should follow this pattern:
<scope>: title<.>